lib/build-parser.js

```diff
@@ -8,20 +8,20 @@ var buildParser = module.exports = function(inFile, outFile) {
     var PEG = require('pegjs');
     var fs = require('fs');
 
-    var parserSource = PEG.buildParser(fs.readFileSync(inFile, 'utf8'), {
+    var parserSource = PEG.generate(fs.readFileSync(inFile, 'utf8'), {
```

Comment:
In the newest version of pegJS (0.10.0), `pegjs.buildParser` was changed to `pegjs.generate`.